requirements.txt

```diff
@@ -1,3 +1,4 @@
+coverage==5.3
```

Comment:
We should have separate `requirements.txt` and `test_requirements.txt` for the production and testing environment accordingly

Reply:
The production or installed environment uses the setup.py `install_requires` field. In public Python packages, there are different ways of doing this, but there's nothing wrong with this particular choice. If you are installing from the requirements.txt, it's because you are trying to develop locally, so you would need everything.